pkg/cmd/cli/cmd/set/volume.go

```diff
@@ -387,6 +401,9 @@ func (v *VolumeOptions) Complete(f *clientcmd.Factory, cmd *cobra.Command, out,
 		}
 		v.AddOpts.ClaimSize = q.String()
 	}
+	if len(v.AddOpts.DefaultMode) == 0 {
+		v.AddOpts.DefaultMode = "644"
```

Comment:
It would be better if the default value in the flag stayed as an empty string so we can identify when a user used the flag.